evm/interchainTokenFactory.js

```diff
@@ -17,7 +17,7 @@ const IInterchainTokenFactory = getContractJSON('IInterchainTokenFactory');
 const IInterchainTokenService = getContractJSON('IInterchainTokenService');
 const IERC20 = getContractJSON('IERC20');
 
-async function processCommand(config, chain, options) {
+async function processCommand(_config, chain, options) {
```

Comment:
why the change to `_config`?

Reply:
Just to remove this warning:
`'config' is declared but its value is never read.`

Reply:
use `_` instead